Sources/WalletConnectHistory/Payloads/NotifyGetNotificationsRequestPayload.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+struct NotifyGetNotificationsRequestPayload: JWTClaimsCodable {
```

Comment:
its history package but has a clear dependency to higher level package Notify with NotifyGetNotificationsRequestPayload